javascript/commons/RankingTable.js

```diff
@@ -3,18 +3,99 @@ liquipedia.rankingTable = {
 	ContentSelector: '[data-ranking-table="content"]',
```

Comment:
not new code, but it's a little bit weird, that this has an uppercase first letter